earth_enterprise/rpms/build.gradle

```diff
@@ -84,9 +70,26 @@ def stagedInstallDir_server_geplaces =
     new File(stagedInstallDir, 'geplaces/opt/google/share/geplaces')
 def stagedInstallDir_server_user_magic =
     new File(stagedInstallDir, 'server/user_magic')
+def stagedInstallDir_server_version =
```

Comment:
The convention seemed to be that all these would start with previously concatenated path names, so I just shortened the end into one word. I could call it stagedInstallDir_server_gee_long_version, but seems too long to me.

Reply:
The purpose of this file is to contain the version string.  It doesn't have much to do with install paths, despite its unfortunate location.

Reply:
"stagedInstallDir" is prepended to all these path variables. It refers to whatever the "installdir" parameter is when running "stage_install". If I changed it here, it would no longer follow the pattern of all the rest. I don't want to change all of them, so I don't want to change this one.

Reply:
You can leave it as it is, but these variables depend on each other, and define the staged install directory hierarchy.  The version file has nothing to do with that hierarchy, and where it is located makes no difference, so the variable doesn't need to reflect that.